### ISSUE TITLE
Plug mappings and more

### ISSUE THREAD

**Issue by jostber** [vimoutliner/vimoutliner#59]
Ref. to post from Israel Chauca previously regarding fixing the code for mappings in VO:

http://comments.gmane.org/gmane.comp.editors.vim.outliner/5516

Is this something we should start working on now?

**Comment by mcepl** [comment]
@noelhenson  any progress on this?

**Comment by mcepl** [comment]
@noelhenson , this probably should be moved to 0.6.0, right?

**Comment by noelhenson** [comment]
Yes. I don't have any free time to implement anything like that at the moment.
